tests/nist/nist_test.sh

```diff
@@ -21,6 +21,60 @@ function test_nist {
 	fi
 }
 
+function test_results_directive {
+	test_dir="$1"
+	run_script="$(cd "${builddir}"; pwd)/run"
+	# make sure the output dir in builddir exists
+	mkdir -p "${builddir}/tests/nist/$test_dir/"
+	local datastream="ind_family_test-datastream.xml"
+
+	local result="${builddir}/tests/nist/${test_dir}/results_directive.results_arf.xml"
+	# the tests of this evaluation dont need to pass, return 0
+	${run_script} $OSCAP xccdf eval --profile "xccdf_gov.nist_profile_family-validation" --results-arf ${result} "${srcdir}/$test_dir/${datastream}" > /dev/null && 0
```

Comment:
Yeah, I was a little bit confused by exit statuses.
But I still don't know why it is working with `&&` if the evaluation of this data stream returns 2.